package.json

```diff
@@ -31,6 +31,7 @@
     "jsdoc": "^3.6.3",
     "mongoose": "^5.7.7",
     "react-router-dom": "^5.1.2",
+    "react-star-rating-component": "^1.4.1",
```

Comment:
this should probably be put in the client/package.json since it's frontend?

Reply:
Just removed it from the package.json file, client/package.json already has it so we should be good